app/views/movies/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1><%= @movie.title %></h1>
+<p><strong>Creation Year:</strong> <%= @movie.creation_year %></p>
+<p><strong>Genre:</strong> <%= @movie.genre %></p>
+
+<h2>Actors:</h2>
+<ul>
+  <% @actors.each do |actor| %>
+    <li><%= actor.name %> - <%= actor.age %> years old</li>
+  <% end %>
+</ul>
+
+<p><strong>Average Age of Actors:</strong> <%= @average_age %></p>
+
+<h2>Add an actor</h2>
+<%= form_with url: "/movies/#{@movie.id}", method: :patch  do |f| %>
```

Comment:
Your page & form look fine, but I think we can use a better HTTP verb here - `PATCH` correlates to an "update" action which you have, but if we're updating a movie, I would expect one of its attributes to change - either its creation_year, genre, or title... Instead, we're actually _creating_ a relationship between this movie and an actor by its ID. Since we're creating a relationship (a row in our join table), this should be a `:post` action instead.